README.md

```diff
@@ -78,7 +78,6 @@ If you have an active interest and you want to get involved, you have real power
 ### Office hours
 
 - Join the [SIG Cluster Lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) Google Group for access to documents and calendars.
-- Participate in the conversations on [Kubernetes Discuss][kubernetes discuss]
```

Comment:
i would not remove the link but instead update to the correct/updated location: https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle

Reply:
@Aayush987 please make the required modifications.

Reply:
Ok Thank you for the suggestion

Reply:
@Aayush987 looks good but can you please squash these commits into one. Three commits are uneccesary.

Reply:
@jayesh-srivastava  I actually dont know how to squash commits. Can you tell me how to do it?

Reply:
that is fine for now @Aayush987 i can squash the commits in the merge but for future PRs you can read https://github.com/kubernetes/community/blob/f8158c06a4293b48f3364fd2af76d5f4a2915f0f/contributors/guide/contributor-cheatsheet/README.md#squashing-commits